python/tests/test_async_utils.py

```diff
@@ -0,0 +1,238 @@
+import asyncio
+
+import pytest
+
+from gen3authz import utils
+
+
+def run(coro):
+    return asyncio.get_event_loop().run_until_complete(coro)
+
+
+def test_basic():
+    async def get():
```

Comment:
nitpick: docstrings with a short description of what you're testing

Reply:
I'd vote against having docstrings in test code - unless they are supposed to be reused. The code itself (module names, function names, parametrized fixtures, variable names, etc) and necessary inline comments must be sufficient to explain what is being tested. If a unit test requires an additional sentence to explain what it does, we may want to split the test down to simpler ones.